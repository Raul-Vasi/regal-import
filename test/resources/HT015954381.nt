<http://d-nb.info/gnd/1032936525> <http://d-nb.info/standards/elementset/gnd#preferredNameEntityForThePerson> _:node18jif4taex1 .
<http://d-nb.info/gnd/1032936525> <http://d-nb.info/standards/elementset/gnd#preferredNameForThePerson> "Winter, Andreas"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/115749519> <http://d-nb.info/standards/elementset/gnd#preferredNameEntityForThePerson> _:node18jif4taex2 .
<http://d-nb.info/gnd/115749519> <http://d-nb.info/standards/elementset/gnd#preferredNameForThePerson> "Staab, Steffen"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/136283675> <http://d-nb.info/standards/elementset/gnd#dateOfBirth> "1978"^^<http://www.w3.org/2001/XMLSchema#gYear> .
<http://d-nb.info/gnd/136283675> <http://d-nb.info/standards/elementset/gnd#preferredNameEntityForThePerson> _:node18jif4taex3 .
<http://d-nb.info/gnd/136283675> <http://d-nb.info/standards/elementset/gnd#preferredNameForThePerson> "Parreiras, Fernando Silva"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/5030229-2> <http://d-nb.info/standards/elementset/gnd#preferredNameForTheCorporateBody> "Institut f\u00FCr Informatik <Koblenz>"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://dewey.info/class/004/2009/08/about.en> <http://www.w3.org/2004/02/skos/core#prefLabel> "Data processing & computer science"@en .
<edoweb:3025500> <http://purl.org/dc/elements/1.1/publisher> "Inst. f\u00FCr Informatik"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/dc/terms/creator> <http://d-nb.info/gnd/1032936525> .
<edoweb:3025500> <http://purl.org/dc/terms/creator> <http://d-nb.info/gnd/115749519> .
<edoweb:3025500> <http://purl.org/dc/terms/creator> <http://d-nb.info/gnd/136283675> .
<edoweb:3025500> <http://purl.org/dc/terms/creator> <http://d-nb.info/gnd/5030229-2> .
<edoweb:3025500> <http://purl.org/dc/terms/hasVersion> <http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=3025501&custom_att_2=simple_viewer> .
<edoweb:3025500> <http://purl.org/dc/terms/isPartOf> <http://lobid.org/resource/Edoweb> .
<edoweb:3025500> <http://purl.org/dc/terms/isPartOf> <http://lobid.org/resource/HT015219238> .
<edoweb:3025500> <http://purl.org/dc/terms/isPartOf> <http://lobid.org/resource/RPB> .
<edoweb:3025500> <http://purl.org/dc/terms/issued> "2007"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/dc/terms/language> <http://id.loc.gov/vocabulary/iso639-2/deu> .
<edoweb:3025500> <http://purl.org/dc/terms/subject> <http://dewey.info/class/004/> .
<edoweb:3025500> <http://purl.org/dc/terms/subject> <http://purl.org/lobid/rpb#n884000> .
<edoweb:3025500> <http://purl.org/dc/terms/title> "TwoUse: Integrating UML models and OWL ontologies"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/lobid/lv#fulltextOnline> <http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=3025501&custom_att_2=simple_viewer> .
<edoweb:3025500> <http://purl.org/lobid/lv#hbzID> "HT015954381"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/ontology/bibo/volume> "Nr. 16/2007"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/vocab/frbr/core#exemplar> <http://lobid.org/item/HT015954381%3ADE-929> .
<edoweb:3025500> <http://rdvocab.info/Elements/placeOfPublication> "Koblenz"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://rdvocab.info/Elements/publicationStatement> "Koblenz; Inst. f\u00FCr Informatik; 2007"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://rdvocab.info/Elements/statementOfResponsibility> "Fernando Silva Parreiras ; Steffen Staatb ; Andreas Winter. Universit\u00E4t Koblenz, Landau, Institut f\u00FCr Informatik, FB 4, Informatik"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://data.archiveshub.ac.uk/def/ArchivalResource> .
<edoweb:3025500> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://iflastandards.info/ns/isbd/terms/mediatype/T1002> .
<edoweb:3025500> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/dc/terms/BibliographicResource> .
<edoweb:3025500> <http://www.w3.org/2002/07/owl#sameAs> <http://hub.culturegraph.org/resource/HBZ-HT015954381> .
<edoweb:3025500> <http://www.w3.org/2007/05/powder-s#describedby> <edoweb:3025500/about> .
<edoweb:3025500> <http://xmlns.com/foaf/0.1/isPrimaryTopicOf> <http://193.30.112.134/F/?func=find-c&ccl_term=IDN%3DHT015954381> .
<edoweb:3025500/about> <http://xmlns.com/foaf/0.1/primaryTopic> <edoweb:3025500> .
<edoweb:3025500> <http://www.umbel.org/specifications/vocabulary#isLike> <http://lobid.org/resource/HT015954381> .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/similarTo> <http://klio.hbz-nrw.de:1801/webclient/MetadataManager?pid=3025500> .
<edoweb:3025500> <http://hbz-nrw.de/regal#hasTransformer> "oaidc" .
<edoweb:3025500> <http://hbz-nrw.de/regal#hasTransformer> "epicur" .
<edoweb:3025500.rdf> <http://www.openarchives.org/ore/terms/describes> <edoweb:3025500> .
<edoweb:3025500.rdf> <http://purl.org/dc/terms/modified> "2014-03-19T11:48:04.440+01:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
<edoweb:3025500.rdf> <http://purl.org/dc/terms/created> "2013-10-18T12:40:19.458+02:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/isDescribedBy> <edoweb:3025500.rdf> .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/similarTo> <http://api.edoweb-test.hbz-nrw.de/fedora/objects/edoweb:3025500> .
<edoweb:3025500> <http://hbz-nrw.de/regal#contentType> "monograph" .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/aggregates> <edoweb:3025503> .
<edoweb:3025500> <http://purl.org/dc/terms/hasPart> <edoweb:3025503> .
