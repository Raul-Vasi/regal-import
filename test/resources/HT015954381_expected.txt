<edoweb:3025500> <http://geni-orca.renci.org/owl/topology.owl#hasURN> "urn:nbn:de:hbz:929:02-edoweb:30255004" .
<edoweb:3025500> <http://hbz-nrw.de/regal#contentType> "monograph" .
<edoweb:3025500> <http://hbz-nrw.de/regal#hasTransformer> "epicur" .
<edoweb:3025500> <http://hbz-nrw.de/regal#hasTransformer> "oaidc" .
<edoweb:3025500> <http://iflastandards.info/ns/isbd/elements/P1004> "TwoUse: Integrating UML models and OWL ontologies"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://iflastandards.info/ns/isbd/elements/P1016> "Koblenz"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://iflastandards.info/ns/isbd/elements/P1017> "Inst. f\u00FCr Informatik"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://lobid.org/vocab/lobid#fulltextOnline> <http://digitool.hbz-nrw.de:1801/webclient/DeliveryManager?pid=3025501&custom_att_2=simple_viewer> .
<edoweb:3025500> <http://purl.org/dc/elements/1.1/creator> <http://d-nb.info/gnd/1032936525> .
<edoweb:3025500> <http://purl.org/dc/elements/1.1/creator> <http://d-nb.info/gnd/115749519> .
<edoweb:3025500> <http://purl.org/dc/elements/1.1/creator> <http://d-nb.info/gnd/136283675> .
<edoweb:3025500> <http://purl.org/dc/terms/hasPart> <edoweb:3025503> .
<edoweb:3025500> <http://purl.org/dc/terms/isPartOf> <http://lobid.org/resource/HT015219238> .
<edoweb:3025500> <http://purl.org/dc/terms/issued> "2007"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/dc/terms/language> <http://id.loc.gov/vocabulary/iso639-2/deu> .
<edoweb:3025500> <http://purl.org/dc/terms/medium> <http://iflastandards.info/ns/isbd/terms/mediatype/T1002> .
<edoweb:3025500> <http://purl.org/dc/terms/subject> <http://dewey.info/class/004/> .
<edoweb:3025500> <http://purl.org/dc/terms/title> "TwoUse: Integrating UML models and OWL ontologies"^^<http://www.w3.org/2001/XMLSchema#string> .
<edoweb:3025500> <http://purl.org/lobid/lv#hbzID> "HT015954381" .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/aggregates> <edoweb:3025503> .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/isDescribedBy> <edoweb:3025500.rdf> .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/similarTo> <http://api.edoweb-rlp.de/fedora/objects/edoweb:3025500> .
<edoweb:3025500> <http://www.openarchives.org/ore/terms/similarTo> <http://klio.hbz-nrw.de:1801/webclient/MetadataManager?pid=3025500> .
<edoweb:3025500> <http://www.umbel.org/specifications/vocabulary#isLike> <http://lobid.org/resource/HT015954381> .
<edoweb:3025500> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/dc/terms/BibliographicResource> .
<edoweb:3025500> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/ontology/bibo/Book> .
<edoweb:3025500> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/vocab/frbr/core#Manifestation> .
<edoweb:3025500> <http://www.w3.org/2007/05/powder-s#describedby> <edoweb:3025500/about> .
<edoweb:3025500> <http://xmlns.com/foaf/0.1/isPrimaryTopicOf> <http://193.30.112.134/F/?func=find-c&ccl_term=IDN%3DHT015954381> .
<edoweb:3025500.rdf> <http://purl.org/dc/terms/creator> <https://github.com/edoweb/regal/> .
<edoweb:3025500.rdf> <http://purl.org/dc/terms/modified> "2013-12-04T18:33:41.427+01:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
<edoweb:3025500.rdf> <http://www.openarchives.org/ore/terms/describes> <edoweb:3025500> .
<http://d-nb.info/gnd/1032936525> <http://d-nb.info/standards/elementset/gnd#preferredNameForThePerson> "Winter, Andreas"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/1032936525> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://d-nb.info/standards/elementset/gnd#DifferentiatedPerson> .
<http://d-nb.info/gnd/115749519> <http://d-nb.info/standards/elementset/gnd#preferredNameForThePerson> "Staab, Steffen"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/115749519> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://d-nb.info/standards/elementset/gnd#UndifferentiatedPerson> .
<http://d-nb.info/gnd/136283675> <http://d-nb.info/standards/elementset/gnd#dateOfBirth> "1978"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/136283675> <http://d-nb.info/standards/elementset/gnd#preferredNameForThePerson> "Parreiras, Fernando Silva"^^<http://www.w3.org/2001/XMLSchema#string> .
<http://d-nb.info/gnd/136283675> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://d-nb.info/standards/elementset/gnd#DifferentiatedPerson> .
<http://dewey.info/class/004/2009/08/about.en> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/2004/02/skos/core#Concept> .
<http://dewey.info/class/004/2009/08/about.en> <http://www.w3.org/2004/02/skos/core#prefLabel> "Data processing & computer science"@en .
